### PR TITLE
Don't use /public path for custom functions with shared runtime

### PR DESCRIPTION
**Pull request by akrantz**
This fixes the manifest so the custom functions metadata (functions.json) and code (functions.js) come directly from the root. It should not try to load from /public.

1. **Do these changes impact any *npm scripts* commands (in package.json)?** (e.g., running 'npm run start')
No

2. **Do these changes impact *VS Code debugging* options (launch.json)?**
No

3. **Do these changes impact *template output*?** (e.g., add/remove file, update file location, update file contents)
No

4. **Do these changes impact *documentation*?** (e.g., a tutorial on https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins)
No

Describe manual testing done.

1. Open VS Code.
2. Start Dev-Server task using Terminal, Run task...
3. Verified in browser that https://localhost:3000/functions.json serves up metadata correctly. Likewise for functions.js and taskpane.html.
4. Set a breakpoint in functions.ts on add() function.
5. Launch Excel Desktop for debugging. 
6. Type "=CONTOSO.ADD(1, 2)" in a cell.
7. Hit the breakpoint in functions.ts.  Press play button to continue.
8. See that the custom function returned 3 which is shown in the cell.

### CODE REVIEW

File: .hintrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "extends": [
+    "development"
+  ],
+  "hints": {
+    "typescript-config/strict": "off"
+  }
+}
```
Comment (millerds) [inline .hintrc]: This doesn't look like it applies to the public path

File: null [hunk 2]
```diff
@@ -0,0 +1 @@
+Error:  Access is denied.
```
Comment (millerds) [inline null]: What's this for?

**Review by millerds — CHANGES_REQUESTED** [review]
Shouldn't we change the webpack config files to not have the public path anymore (they are only there to support functions)?

**Comment by millerds** [comment]
Just saw the update . . . sorry about the extra comments.   Still think the webpack config files should be updated.

**Comment by akrantz** [comment]
Is your question whether to delete the `publicPath: "/public",` in the devServer part of webpack config?

```
    devServer: {
      static: {
        directory: path.join(__dirname, "dist"),
        publicPath: "/public",
      },
```

I can delete it, although I don't think it makes a difference really.

**Comment by akrantz** [comment]
I pushed a commit that removes devServer.static.

**Comment by millerds** [comment]
Thanks.  What about the test manifest and webpack config?  We should make sure it test the same way it's used.

**Comment by akrantz** [comment]
Updated.

**Pull request merged**